quickstarts/install-odf.yaml

```diff
@@ -7,25 +7,25 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
 spec:
-  displayName: Install the OpenShift Container Storage Operator
+  displayName: Install the OpenShift Data Foundation
   durationMinutes: 5
   icon: data:image/svg+xml;base64,PHN2ZyBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAxMDAgMTAwIiBoZWlnaHQ9IjEwMCIgdmlld0JveD0iMCAwIDEwMCAxMDAiIHdpZHRoPSIxMDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTY2LjcgNTUuOGM2LjYgMCAxNi4xLTEuNCAxNi4xLTkuMiAwLS42IDAtMS4yLS4yLTEuOGwtMy45LTE3Yy0uOS0zLjctMS43LTUuNC04LjMtOC43LTUuMS0yLjYtMTYuMi02LjktMTkuNS02LjktMy4xIDAtNCA0LTcuNiA0LTMuNSAwLTYuMS0yLjktOS40LTIuOS0zLjIgMC01LjIgMi4xLTYuOCA2LjYgMCAwLTQuNCAxMi41LTUgMTQuMy0uMS4zLS4xLjctLjEgMSAuMSA0LjcgMTkuMiAyMC42IDQ0LjcgMjAuNm0xNy4xLTZjLjkgNC4zLjkgNC44LjkgNS4zIDAgNy40LTguMyAxMS40LTE5LjEgMTEuNC0yNC42IDAtNDYuMS0xNC40LTQ2LjEtMjMuOSAwLTEuMy4zLTIuNi44LTMuOS04LjkuNS0yMC4zIDIuMS0yMC4zIDEyLjIgMCAxNi41IDM5LjIgMzYuOSA3MC4yIDM2LjkgMjMuOCAwIDI5LjgtMTAuNyAyOS44LTE5LjIgMC02LjctNS44LTE0LjMtMTYuMi0xOC44IiBmaWxsPSIjZWQxYzI0Ii8+PHBhdGggZD0ibTgzLjggNDkuOGMuOSA0LjMuOSA0LjguOSA1LjMgMCA3LjQtOC4zIDExLjQtMTkuMSAxMS40LTI0LjYgMC00Ni4xLTE0LjQtNDYuMS0yMy45IDAtMS4zLjMtMi42LjgtMy45bDEuOS00LjhjLS4xLjMtLjEuNy0uMSAxIDAgNC44IDE5LjEgMjAuNyA0NC43IDIwLjcgNi42IDAgMTYuMS0xLjQgMTYuMS05LjIgMC0uNiAwLTEuMi0uMi0xLjh6IiBmaWxsPSIjMDEwMTAxIi8+PC9zdmc+
-  description: Install the OpenShift Container Storage (OCS) operator and create a storage cluster.
-  introduction: Red Hat OpenShift® Container Storage is persistent software-defined storage integrated with and optimized for Red Hat OpenShift Container Platform. Dynamic, stateful, and highly available container-native storage can be provisioned and de-provisioned on demand as an integral part of the OpenShift administrator console.
+  description: Install the OpenShift Data Foundation (ODF) operator and create a storage system.
+  introduction: Red Hat OpenShift® Data Foundation is persistent software-defined storage integrated with and optimized for Red Hat OpenShift Container Platform. Dynamic, stateful, and highly available container-native storage can be provisioned and de-provisioned on demand as an integral part of the OpenShift administrator console.
   tasks:
-  - title: Install the OpenShift Container Storage operator
+  - title: Install the OpenShift Data Foundation
     description: |- 
       The OperatorHub is where you can find a catalog of available Operators to install on your cluster.
       
-      To install OpenShift Container Storage, follow these steps:
+      To install OpenShift Data Foundation, follow these steps:
 
       1. Enter the administrator perspective: In the main navigation, click the dropdown menu and select Administrator.
```

Comment:
I'd suggest using this, which will enable discovery of the perspective switcher.  @beanh66 i'm not sure if this wording works, so you may want to verify!

Enter the administrator perspective: In the main navigation, click the [perspective switcher]{{highlight qs-perspective-switcher}} and select Administrator.

Reply:
Updated and added the proper perspective switchers

Reply:
Added